python/railjson_generator/justfile

```
@@ -0,0 +1,33 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Generate small infra
+run:
+    mkdir -p small_infra_out
+    poetry run scripts/small_infra.py small_infra_out
+
+# Install required dependencies
+install:
+    poetry install
```

Do we want to default to `--all-extras`?

```suggestion
    poetry install --all-extras
```

---

There are no extra dependencies, so I suppose it’s the same issue as with cargo: should we consider that its defaults are _in general_ bad, and we should change them, even if they don’t apply?

---

As discussed, I added --all-extras